TowerForge/TowerForge/GameModule/PowerUps/InvulnerabilityPowerUp.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  InvulnerabilityPowerUp.swift
+//  TowerForge
+//
+//  Created by Keith Gan on 31/3/24.
+//
+
+class InvulnerabilityPowerUp: EventTransformation {
+    func transformEvent(event: TFEvent) -> TFEvent {
```

Comment:
I think that we probably need to be able remove this transform somehow or for the powerup to expire from time

Reply:
will add in a follow-up PR